### ISSUE TITLE
Keystone-next Next.js Embedded tutorial, Admin UI doesn't launch and throws error

### ISSUE THREAD

**Issue by ADSmith-0** [keystonejs/keystone#6373]
# Bug report

Please use this form for bug reports only. If you have a question, feature request or want to chat with us, please select the appropriate option here - https://github.com/keystonejs/keystone/issues/new/choose

## Describe the bug

When following the [embed tutorial](https://keystonejs.com/docs/walkthroughs/embedded-mode-with-sqlite-nextjs#start-all-the-things), I'm getting the error: 
```
✨ Generated GraphQL and Prisma schemas
event - compiled successfully
✨ Starting Keystone
Error: listen EADDRINUSE: address already in use :::8000
    at Server.setupListenHandle [as _listen2] (net.js:1318:16)
    at listenInCluster (net.js:1366:12)
    at Server.listen (net.js:1452:7)
    at Function.listen (C:\Users\ADS\Desktop\Work\prmte-me\node_modules\express\lib\application.js:618:24)
    at dev (C:\Users\ADS\Desktop\Work\prmte-me\node_modules\@keystone-next\keystone\scripts\dist\keystone.cjs.dev.js:359:22)
    at cli (C:\Users\ADS\Desktop\Work\prmte-me\node_modules\@keystone-next\keystone\scripts\dist\keystone.cjs.dev.js:659:12)
    at Object.<anonymous> (C:\Users\ADS\Desktop\Work\prmte-me\node_modules\@keystone-next\keystone\scripts\dist\keystone.cjs.dev.js:669:1)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
```

It says the address is already in use. I've used .env to change the port to 4500 and gotten the same error of
`Error: listen EADDRINUSE: address already in use :::4500`

When I navigate to localhost:8000 I get: 
```
Server Error
Error: Cannot find module 'next/dist/next-server/lib/constants.js'
```

I've tried a clean install and following the tutorial again, and the same error has occured

I also went into netstat to kill any tasks that might be on the same port, but found none, it appears this only occurs when I start up the server with `yarn dev`

## To Reproduce

Steps to reproduce the behaviour. Please provide code snippets or a repository:

1. [Following this tutorial](https://keystonejs.com/docs/walkthroughs/embedded-mode-with-sqlite-nextjs)
2. My repository of the code with the error can be [found here](https://github.com/ADSmith-0/prmte-me)

## Expected behaviour
Expected the Admin UI to launch and be working on port 8000

## Screenshots

N/A

## System information

- OS: Windows
- Browser (if applies): Firefox, the error occurs on Chrome as well

## Additional context

N/A

**Comment by tmadison-gpsw** [comment]
I am getting an issue with Next.js "10.2.3" with the embedded tutorial as well (https://keystonejs.com/docs/walkthroughs/embedded-mode-with-sqlite-nextjs)

<img width="973" alt="Screen Shot 2021-08-19 at 6 31 31 PM" src="https://user-images.githubusercontent.com/16724169/130165072-8ff0dbae-0d6f-4cbc-860b-75508e8a2ba9.png">

**Comment by bladey** [comment]
Hi @ADSmith-0 and @tmadison-gpsw,

Thanks for reporting these issues.

@ADSmith-0 Next.js `11.x` is not compatible with the embedded version of Keystone until we upgrade Keystone's Next.js internal version to `11.x`. I've since adjusted the guide to temporarily work off version `10.x` with an alternative repo to clone off.

@tmadison-gpsw You're right, the filter was out of date to align with our new GraphQL API. I've also adjusted this:

Previously:

`where: { slug: params!.slug as string },`

Now:

`where: { slug: { equals: params!.slug as string } },`
